gradle.properties

```diff
@@ -1 +1,2 @@
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+kotlin.daemon.jvmargs=-Xmx8049m
```

Comment:
Do we need this again?

Reply:
yes. I have added bunch more icons and the build started to fail again. before I forgot `m` at the end, so it had no effect. 

Now it will be fine.